layouts/_default/baseof.html

```diff
@@ -83,27 +83,7 @@
 
     <header role="banner">
       {{ partial "fip.html" . }}
-
-      {{ block "title" . }}
       {{partial "menu.html" . }}
-      <div class="page-banner">
-        <div class="container">
-          <div class="page-banner--title">
-            {{ $title := i18n .Params.title }}
-            {{ with $title }}
-            <h1>{{ $title }}</h1>
```

Comment:
This is the base template for most pages, does this end up removing the `h1` entry from most of them?

Reply:
I reverted this portion. This has to do with the top banner's title, so instead of including this in the job postings commit, this can be it's seperate issue. I don't know if this will further cause accessibility issues.